offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/update/UpdatePersonView.groovy

```diff
@@ -55,9 +60,14 @@ class UpdatePersonView extends CreatePersonView {
         //add a grid
         affiliations = new Grid<>()
         generateAffiliationGrid()
+        HorizontalLayout gridHeader = new HorizontalLayout()
         Label currentAffiliationLabel = new Label("Current Affiliations")
-        affiliations.setSelectionMode(Grid.SelectionMode.NONE)
-        defaultContent.addComponent(currentAffiliationLabel, 3)
+        removeSelectedAffiliationsButton = new Button("Remove affiliations")
+        removeSelectedAffiliationsButton.setEnabled(false)
+        removeSelectedAffiliationsButton.setIcon(VaadinIcons.TRASH)
+        affiliations.setSelectionMode(Grid.SelectionMode.MULTI)
+        gridHeader.addComponents(currentAffiliationLabel, removeSelectedAffiliationsButton)
+        defaultContent.addComponent(gridHeader, 3)
```

Comment:
why is that in here? I thought this is from another PR?

Reply:
👀 nice spot, I must have accidently checked out from the feature branch instead of development